tests/foreman/cli/test_contentviewfilter.py

```diff
@@ -37,7 +37,7 @@ def setUpClass(cls):
         })
 
     @tier1
-    def test_create_cvf_different_names(self):
+    def test_positive_create_with_name_by_cv_id(self):
```

Comment:
@sghai - @abalakh is right, we are following the naming convention from https://github.com/SatelliteQE/robottelo/issues/3083#issuecomment-161375918.  And use of additional verbs like `attach` or `assign` is not required if it makes sense without them.